module/gridss.nf

```diff
@@ -56,3 +56,62 @@ process preprocess_BAM_GRIDSS {
             ${sample_bam}
         """
     }
+
+process run_assembly_GRIDSS {
+    container params.docker_image_gridss
+
+    publishDir "${params.workflow_output_dir}/output",
+        pattern: "${tumor_id}.assembly.bam",
+        mode: "copy",
+        saveAs: {
+            "${output_filename}_${sanitize_string(file(it).getName().replace("${tumor_id}.", ""))}"
+            }
```

Comment:
**comment (blocking)**: What's in this BAM file? It seems like it should be an intermediate file rather than an output

Reply:
`assembly.bam` contains breakend assemblies that are used in conjunction with input T/N BAMs for variant calling.

The official GRIDSS documentation hasn't described the `assembly.bam` as an intermediate but it has described `assembly.sv.bam.gridss.working` as an intermediate.

`assembly.bam` is explicitly defined as a required parameter for step 3 in GRIDSS which is variant calling. For this reason I thought I'd put this in the output dir.

Reply:
Hmm from a pipeline perspective, the BAM should then be intermediate since it's used between steps within GRIDSS but not the actual output of GRIDSS (the variant calls). If the BAM is only intended for use for the third step in GRIDSS, we don't need to be saving the BAM every time we run the pipeline

Reply:
I was wondering the same. The intermediate BAM files however can be used for SV visualization using IGV, so I'm thinking we add documentation to README suggesting users to set `save_intermediate_files = true` if they want these intermediate BAMs for visualization or any downstream steps.

Reply:
Sure, that's fine to document the option in the README for users who might want it